src/OSPSuite.SimModelNative/src/ProductFormula.cpp

```diff
@@ -116,7 +116,7 @@ void ProductFormula::DE_Jacobian (double * * jacobian, const double * y, const d
 				prod *= _multiplierFormulas[nr] -> DE_Compute(y, time, USE_SCALEFACTOR);
 
 				if (prod == 0.0)
-					continue;
```

Comment:
Here

Reply:
Somehow I'm confused: GitHub shows me as diff this:
![grafik](https://user-images.githubusercontent.com/25061876/157903289-64176651-10e4-4e6c-9e13-f8110901ee9c.png)

Also I already fixed this, which looks in master like this now:
https://github.com/Open-Systems-Pharmacology/OSPSuite.SimModel/blob/41eb2180fdbeec59e8a343d3f60a906628244c9b/src/OSPSuite.SimModelNative/src/ProductFormula.cpp#L102-L129